puppet/modules/jenkins_job_builder/files/centos.org/centos_jenkins.ini.example

```diff
@@ -0,0 +1,10 @@
+[job_builder]
```

Comment:
Is there a reason this is named `.ini.example`? Why not just `.ini`?

Reply:
I did it because we wanted to avoid having it before (see we have a .gitignore with .ini files). I think it doesn't make sense to keep it and we can instead just have a .ini file.

Reply:
That is only `jenkins_job.ini` though I'm not sure why @ehelms added that in 9fd0ec78e9072f7fa3d1afd0340695d3f5c6191a.